src/sentry/testutils/helpers/options.py

```diff
@@ -15,18 +17,27 @@ def override_options(options):
     from django.conf import settings
 
     from sentry.options import default_manager
+    from sentry.options.manager import OptionsManager
 
     wrapped = default_manager.store.get
+    original_lookup = OptionsManager.lookup_key
 
     def new_get(key, **kwargs):
         try:
             return options[key.name]
         except KeyError:
             return wrapped(key, **kwargs)
 
+    def new_lookup(self: OptionsManager, key):
+        if key in options:
+            return self.make_key(key, lambda: "", Any, 1 << 0, 0, 0, None)
+        else:
+            return original_lookup(self, key)
+
     # Patch options into SENTRY_OPTIONS as well
     new_options = settings.SENTRY_OPTIONS.copy()
     new_options.update(options)
     with override_settings(SENTRY_OPTIONS=new_options):
         with patch.object(default_manager.store, "get", side_effect=new_get):
-            yield
+            with patch("sentry.options.OptionsManager.lookup_key", new=new_lookup):
```

Comment:
You don't need to nest these ctx managers you can actually just use `,`

Reply:
Good catch, will make the change